contracts/dao/nopeDAO/NopeDAO.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Governor, IGovernor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+import {GovernorVotesComp, IERC165} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
+import {CoreRef} from "../../refs/CoreRef.sol";
+import {TribeRoles} from "../../core/TribeRoles.sol";
+import {GovernorQuickReaction} from "./GovernorQuickReaction.sol";
+
+contract NopeDAO is
+    Governor,
+    GovernorSettings,
+    GovernorVotesComp,
+    GovernorQuickReaction,
+    CoreRef
```

Comment:
This architecture is really interesting. I thought the NopeDAO would have been created as an opinionated DAO that could only call the cancel function of another DAO, but this is a way better architecture and keeps backwards compatibility with all the existing DAO infrastructure.